crates/sui-graphql-rpc/src/types/object.rs

```diff
@@ -88,6 +98,42 @@ impl Object {
         Some(BigInt::from(self.native.storage_rebate))
     }
 
+    async fn display(&self, ctx: &Context<'_>) -> Result<Option<Vec<DisplayEntry>>> {
```

Comment:
rn im inclined to keep Vec<DisplayEntry> as a vector instead of refactoring into a connection, because each entry in the display necessitates the entire MoveStruct to fill in the corresponding template

on the other hand, it would be a bit annoying to have some connections here and some vecs there

Reply:
Same, not sure what the best approach here is either. Even though there are bounds on Display I could probably create a really nasty template string. Dumb question but - I know today the fullnodes are responsible for this processing, but why not offloading it onto the client and sdk?